frontend/src/data/Breakdowns.ts

```diff
@@ -53,16 +53,21 @@ export class Breakdowns {
 
   filterFips?: string;
 
+  // Whether or not a we should include a "Total" value in response which sums total population
```

Comment:
Replace "sums total population" with "sums the total value for the metric across the breakdown"